MAINTAINERS.md

```diff
@@ -0,0 +1 @@
+MAINTAINERS
```

Comment:
What does this do?

Reply:
It's a symbolic link

Reply:
Why do we need a symbolic link for the MAINTAINERS?

Reply:
As discussed previously we want to move to markdown which is done on this PR but we can't just delete the existing file because each repo points to it.
Until we modify each repo we keep both (hopefully as part of the security process PRs we'll update this link as well)